ledger/ledgercore/accountdata.go

```diff
@@ -69,6 +69,7 @@ type VotingData struct {
 type OnlineAccountData struct {
 	MicroAlgosWithRewards basics.MicroAlgos
 	VotingData
+	IncentiveEligible bool
```

Comment:
I believe we should update `AccountData.OnlineAccountData` to include this field?

Reply:
Probably! Will look.

Reply:
The method
```
func (u AccountData) OnlineAccountData() OnlineAccountData {
```
appears to be only used in tests outside of `basics` package. TODO: move to basics/testing after this is merged.

Reply:
https://github.com/algorand/go-algorand/pull/5757/commits/fd1c88b83ce25233d7c59b4863c5efb2c8e0e9de